src/validateEmail.js

```diff
@@ -0,0 +1,17 @@
+export default validateEmail
+
+const regex = new RegExp(['^(([^<>()[]\\.,;:\\s@"]+(\\.[^<>(),[]\\.,;:\\s@"]+)*)',
```

Comment:
Ah yes, so the thing that's breaking here is the [max-len](http://eslint.org/docs/rules/max-len) rule from ESLint. Most of the time we want to keep our lines from getting too wide, but in some cases, it can't really be helped. So you'll just have to disable it for this line. To do this, it'll look like this:

```javascript
const regex = new RegExp(/* your regex */) // eslint-disable-line max-len
```